diff --git a/setup.py b/setup.py
index 2dcf32a..d5d54df 100755
--- a/setup.py
+++ b/setup.py
@@ -127,8 +127,6 @@ setup(name="authentic2",
           'django-mellon',
           'ldaptools',
           'django-jsonfield',
-          'jwcrypto>=0.3.1,<1',
-          'cryptography',
           'XStatic-jQuery',
           'XStatic-jquery-ui',
       ],
@@ -162,11 +160,9 @@ setup(name="authentic2",
           'authentic2.plugin': [
               'authentic2-auth-ssl = authentic2.auth2_auth.auth2_ssl:Plugin',
               'authentic2-auth-saml = authentic2_auth_saml:Plugin',
-              'authentic2-auth-oidc = authentic2_auth_oidc:Plugin',
               'authentic2-idp-saml2 = authentic2.idp.saml:Plugin',
               'authentic2-idp-openid = authentic2_idp_openid:Plugin',
               'authentic2-idp-cas = authentic2_idp_cas:Plugin',
-              'authentic2-idp-oidc = authentic2_idp_oidc:Plugin',
               'authentic2-provisionning-ldap = authentic2_provisionning_ldap:Plugin',
           ],
       })
